src/zenml/cli/artifact.py

```diff
@@ -50,6 +49,80 @@ def list_artifacts(**kwargs: Any) -> None:
 
     cli_utils.print_pydantic_models(
         artifacts,
+        exclude_columns=["created", "updated", "has_custom_name"],
+    )
+
+
+@artifact.command("update", help="Update an artifact.")
+@click.argument("artifact_name_or_id")
+@click.option(
+    "--name",
+    "-n",
+    type=str,
+    help="New name of the artifact.",
+)
+@click.option(
+    "--tag",
+    "-t",
+    type=str,
+    multiple=True,
+    help="Tags to add to the artifact.",
```

Comment:
Is it worth adding something to the CLI __init__.py docs (or in the docstring?) explaining exactly how you should format / add the tags? The help text suggests you can add 'tags', but as far as I understand it you add each tag separately / individually? Is a bit unclear.

Reply:
In the CLI init we have this:

> If you would like to rename an artifact or adjust the tags of an artifact or
artifact version, use the corresponding `update` command:
```bash
zenml artifact update <NAME> -n <NEW_NAME>
zenml artifact update <NAME> -t <TAG1> -t <TAG2> -r <TAG_TO_REMOVE>
zenml artifact version update <NAME> -v <VERSION> -t <TAG1> -t <TAG2> -r <TAG_TO_REMOVE>
```

Isn't that clear enough?

Reply:
That is indeed clear enough in the CLI init. But could you add an equivalent example to the relevant docstrings as well?

Reply:
Yes sure 👍